Sources/Sentry/SentryScope.m

```diff
@@ -7,6 +7,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+NSInteger const defaultMaxBreadcrumbsCap = 250;
```

Comment:
unified API docs say `100`

Reply:
Where do we say this?

Reply:
https://docs.sentry.io/error-reporting/configuration/?platform=python#max-breadcrumbs

Reply:
Ahh ok that's the default value and it is 100
https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/SentryOptions.m#L90

This value really is the upper hard cap limit even if you put `1000` in maxBreadcrumbs is caps at 250